### ISSUE TITLE
Link in README is dead

### ISSUE THREAD

**Issue by fmayer** [LinuxStandardBase/lsb#23]
http://bzr.linuxfoundation.org/loggerhead/lsb/  is a dead link in the README

**Comment by mwichmann** [comment]
Yes, afaik, bzr.linuxfoundation.org isn't coming back. Some materials are duplicated in branches here, converted to git; the rest have archival copies and could be pushed to github if there's any particular interest.
